pkg/scheduler/api/job_info.go

```diff
@@ -711,21 +715,87 @@ func (ji *JobInfo) PendingBestEffortTaskNum() int32 {
 	return int32(count)
 }
 
+// CheckJobNeedContinueAllocating checks whether it can continue to allocate for current job
+// there are two cases to continue:
+// 1. job's total allocable number meet its minAvailable
```

Comment:
Why need to continue allocate when gang is meet?

Reply:
It is not allocated, but allocable, which means the left pods may be can be predicated succesfully and meet the gang-scheduling.